pinot-integration-tests/src/test/java/org/apache/pinot/compat/tests/SegmentOp.java

```diff
@@ -36,15 +57,23 @@
  */
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class SegmentOp extends BaseOp {
+  private static final Logger LOGGER = LoggerFactory.getLogger(SegmentOp.class);
+  private static final FileFormat DEFAULT_FILE_FORMAT = FileFormat.CSV;
+  private static final String STATE_ONLINE = "ONLINE";
+  private static final int DEFAULT_MAX_SLEEP_TIME_MS = 30000;
+  private static final int DEFAULT_SLEEP_INTERVAL_MS = 200;
+
   public enum Op {
-    UPLOAD,
-    DELETE
+    UPLOAD, DELETE
```

Comment:
nit:  I prefer one per line for readability

Reply:
Intellij pinot default code formatter moves to a single line may have to consider making changes to codestyle-intellij.xml to take care of this issue.